tools/alpine/Makefile

```diff
@@ -9,25 +9,25 @@ default: push
 show-tag:
 	@sed -n -e '1s/# \(.*\/.*:[0-9a-f]\{40\}\)/\1/p;q' versions
 
-hash: Dockerfile Makefile packages
+iid: Dockerfile Makefile packages
```

Comment:
Maybe do the `iid` change in a different PR, as want the containerd stuff merged ASAP as will cause runc version sync issues